src/position/test/position.spec.js

```diff
@@ -16,18 +16,26 @@ describe('position elements', function () {
     $position = _$position_;
   }));
   beforeEach(function () {
-    this.addMatchers({
-      toBePositionedAt: function(top, left) {
-        this.message = function() {
-          return 'Expected "('  + this.actual.top + ', ' + this.actual.left +  ')" to be positioned at (' + top + ', ' + left + ')';
+    jasmine.addMatchers({
+      toBePositionedAt: function(util, customEqualityTesters) {
+        return {
+          compare: function(actual, top, left) {
+            var result = {
+              pass: util.equals(actual.top, top, customEqualityTesters) &&
+                      util.equals(actual.left, left, customEqualityTesters)
```

Comment:
Probably shouldn't convert these to `util.equals` either. Since they're supposed to be normal equality comparisons. I'm not sure if there's a good reason to use `util.equals` in these cases.